samples/Mvc.Client/Startup.cs

```diff
@@ -46,16 +46,14 @@ public void ConfigureServices(IServiceCollection services)
             options.Scope.Add("user:email");
         })
 
-        /*
-        .AddApple(options =>
+        /*.AddApple(options =>
```

Comment:
The changes in this file look like leftovers from your local testing. Please revert them.